assets/src/components/app.tsx

```diff
@@ -52,7 +52,9 @@ export const AppRoutes = () => {
   }, [path, setPath])
 
   const vehiclesByRouteIdNeeded =
-    openView === OpenView.Late || location.pathname === "/"
+    openView === OpenView.Late ||
```

Comment:
This seems kind of brittle (it was before too, but now that there are three conditions, it seems especially so), but I don't have any great ideas for how to de-brittle-ify it. I wish there was some way that the view itself could tell `app` "I need `vehiclesByRouteId`".

Possibly having `useVehicles` be called within the page components themselves? I had a thought that that wouldn't be possible because of the socket, but the socket should be accessible within the page components by way of `useContext`.

This is a non-blocking comment, and probably shouldn't be addressed in this PR anyway, since that would run the risk of creating PR bloat.